src/state/nonces/index.test.ts

```diff
@@ -0,0 +1,196 @@
+const TEST_ADDRESS_1 = '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266';
```

Comment:
test are failing cause we're pulling address from redux as a fallback, gonna pull that out